src/cf.data.pre

```diff
@@ -6969,6 +6969,24 @@ DOC_START
 	varying objects not intended for caching to get cached.
 DOC_END
 
+NAME: ignore_vary
+COMMENT: on|off
+TYPE: onoff
+LOC: Config.onoff.ignore_vary
+DEFAULT: off
+DOC_START
+	Some HTTP servers send a Vary header even though the clients
+	never send requests with different request headers.  This
+	option causes Squid to completely ignore the Vary header and
+	go ahead and cache the response using normal rules.
```

Comment:
If you agree with the following assertion, please adjust PR description to explain that the proposed `ignore_vary on` differs from the already supported `reply_header_access Vary deny all` because the new feature is applied to all responses, while existing `reply_header_access` directive only applies to responses _sent_ by Squid.